test/python/pulse/test_instruction_schedule_map.py

```diff
@@ -16,13 +16,15 @@
 import numpy as np
 
 import qiskit.pulse.library as library
-from qiskit.test import QiskitTestCase
```

Comment:
I think some of these updated as a result of running `make lint` or `make style`